api-reference/10 UI Components/dxCardView/1 Configuration/scrolling/useNative.md

```diff
@@ -0,0 +1,30 @@
+---
+id: dxCardView.Options.scrolling.useNative
+type: Boolean | Enums.Mode
+default: 'auto'
+---
+---
+##### shortDescription
+Specifies whether CardView should use native or simulated scrolling.
+
+---
+This property can have one of the following values:
+
+<table class="dx-table">
+    <tr>
+        <th>scrolling.useNative</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td><i>"auto"</i></td>
+        <td>Native scrolling on all platforms, except non-Mac desktops.</td>
```

Comment:
Minor
I know that this is copied from scrollable, but "except non-Mac desktops" is hard to read because of double negation
Related to same description in other places